src/main/java/org/pesmypetcare/webservice/dao/communitymanager/ForumDaoImpl.java

```diff
@@ -34,30 +39,35 @@
     private static final String TAGS_FIELD = "tags";
     private static final String FORUMS_FIELD = "forums";
     private static final String FORUM_FIELD = "forum";
-    private final DateTimeFormatter timeFormatter;
+    private static final String LIKED_BY_FIELD = "likedBy";
+    private FirebaseMessaging firebaseMessaging;
     @Autowired
     private GroupDao groupDao;
     @Autowired
+    private UserDao userDao;
+    @Autowired
+    private StorageDao storageDao;
+    @Autowired
     private FirestoreDocument documentAdapter;
     @Autowired
     private FirestoreCollection collectionAdapter;
 
     public ForumDaoImpl() {
-        timeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss", new Locale("es", "ES"));
+        firebaseMessaging = FirebaseFactory.getInstance().getFirebaseMessaging();
     }
 
     @Override
     public boolean forumNameInUse(String parentGroup, String forumName) throws DatabaseAccessException {
-        return documentAdapter.documentExists(Path.ofDocument(Collections.forumsNames, parentGroup, forumName));
+        return documentAdapter.documentExists(Path.ofDocument(Collections.forum_names, parentGroup, forumName));
     }
 
     @Override
     public void createForum(String parentGroup, ForumEntity forumEntity)
         throws DatabaseAccessException, DocumentException {
-        forumEntity.setCreationDate(timeFormatter.format(LocalDateTime.now()));
+        forumEntity.setCreationDate(UTCLocalConverter.getCurrentUTC());
         WriteBatch batch = documentAdapter.batch();
         String parentId = documentAdapter
-            .getStringFromDocument(Path.ofDocument(Collections.groupsNames, parentGroup), "group");
+            .getStringFromDocument(Path.ofDocument(Collections.groups_names, parentGroup), "group");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "group" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5534760)